components/sup-protocol/Cargo.toml

```diff
@@ -17,6 +17,7 @@ prost-derive = "*"
 rand = "*"
 serde = "*"
 serde_derive = "*"
+tempfile = "*"
```

Comment:
Moving this to `dev-dependencies` avoids adding it when building for a non-dev, non-test environment. See https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies

Reply:
Good call. I have moved this under `dev-dependencies`.